quinn-proto/src/config.rs

```diff
@@ -434,6 +434,11 @@ where
     /// Improves behavior for clients that move between different internet connections or suffer NAT
     /// rebinding. Enabled by default.
     pub(crate) migration: bool,
+
+    /// Whether to use the local IP address which was used when receiving the initial
```

Comment:
Nit on docstring: we prefer to have a short first line (no period at the end, since it'll be a sentence fragment), then extended explanation after an empty line. This gets the optimal result in rustdoc (where the initial line is used in lists of items).